spec/swagger.yaml

```diff
@@ -2385,14 +2385,15 @@ paths:
 #
 # Issues API endpoints
 #
-  /issues-api/v2/dictionary/issue-types:
+  '/issues-api/v2/dictionary/issue-types':
     get:
-      summary: List all issue types
+      summary: Get Issue Types (DEPRECATED)
```

Comment:
i dont think the first API listed should be a deprecated API. This should be moved to the bottom of the APi's for Issues